core_impl/src/test/java/io/opencensus/stats/StatsContextTest.java

```diff
@@ -111,75 +111,45 @@ public void testWithComposed() {
-//        viewManager.getView(
-//            RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
-//    afterViewData.match(
-//        new Function<DistributionViewData, Void>() {
-//          @Override
-//          public Void apply(DistributionViewData view) {
-//            assertThat(view.getDistributionAggregates()).hasSize(1);
-//            DistributionAggregate agg = view.getDistributionAggregates().get(0);
-//            assertThat(agg.getTags())
-//                .containsExactly(
-//                    Tag.create(
-//                        RpcMeasureConstants.RPC_CLIENT_METHOD, TagValue.create("myMethod")));
-//            assertThat(agg.getCount()).isEqualTo(1);
-//            assertThat(agg.getMean()).isWithin(TOLERANCE).of(5.1);
-//            return null;
-//          }
-//        },
-//        new Function<IntervalViewData, Void>() {
-//          @Override
-//          public Void apply(IntervalViewData view) {
-//            fail("Expected a DistributionViewData");
-//            return null;
-//          }
-//        });
-//  }
-//
-//  @Test
-//  public void testRecordLong() {
-//    MeasureLong measure = MeasureLong.create("long measure", "description", "1");
-//    viewManager.registerView(
-//        View.DistributionView.create(
-//            View.Name.create("name"),
-//            "description",
-//            measure,
-//            DistributionAggregation.create(),
-//            Arrays.asList(K1)));
-//    thrown.expect(UnsupportedOperationException.class);
-//    defaultStatsContext.with(K1, V1).record(MeasureMap.builder().set(measure,1L).build());
-//  }
+  // The main tests for stats recording are in ViewManagerImplTest.
+  @Test
+  public void testRecordDouble() {
+    viewManager.registerView(RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
+    ViewData beforeViewData =
+        viewManager.getView(RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
+    assertThat(beforeViewData.getAggregationMap()).isEmpty();
+    StatsContext context =
+        defaultStatsContext.with(
+            RpcMeasureConstants.RPC_CLIENT_METHOD, TagValue.create("myMethod"));
+    MeasureMap measurements =
+        MeasureMap.builder()
+            .set(RpcMeasureConstants.RPC_CLIENT_ROUNDTRIP_LATENCY, 5.1).build();
+    context.record(measurements);
+    ViewData afterViewData =
+        viewManager.getView(
+            RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
+    assertThat(afterViewData.getAggregationMap()).hasSize(1);
+    for (Entry<List<TagValue>, List<AggregationData>> entry :
+        afterViewData.getAggregationMap().entrySet()) {
+      assertThat(entry.getKey()).containsExactly(TagValue.create("myMethod"));
+      assertThat(entry.getValue()).hasSize(3);
+      for (AggregationData aggregate : entry.getValue()) {
+        StatsTestUtil.assertAggregationDataEquals(aggregate, 5.1, 1L,
+            StatsTestUtil.removeTrailingZeros(0, 0, 0, 0, 0, 0, 1), null, null, null, null,
```

Comment:
Correct, I've added such a method `assertAggregationDataListEquals()` in my next PR #493 : https://github.com/census-instrumentation/opencensus-java/pull/493/commits/72ed9e339f9aade1ba02565b19f5799f4bfbf33b#diff-c24b66b69d113e9ee0e6ddcaea2cf739R200

 I'll add a TODO here and fix this along with other changes in PR #493 .